server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/WebSocketRoutes.scala

```diff
@@ -79,7 +80,12 @@ class WebSocketRoutes @Inject() (@Named(InjectionKeys.RFC_8621) val authenticato
                                  emailChangeRepository: EmailChangeRepository,
                                  pushSerializer: PushSerializer,
                                  typeStateFactory: TypeStateFactory,
-                                 delegationStore: DelegationStore) extends JMAPRoutes {
+                                 delegationStore: DelegationStore,
+                                 metricFactory: MetricFactory,
+                                 gaugeRegistry: GaugeRegistry) extends JMAPRoutes {
+  private val openingConnectionsMetric: Metric = metricFactory.generate("jmap_websocket_opening_connections")
```

Comment:
```suggestion
  private val openingConnectionsMetric: Metric = metricFactory.generate("jmap_websocket_opened_connections")
```

Reply:
Do you mean opened connections of all time, or just opened connections at the moment?